docs/source/Tutorials/custom.rst

```diff
@@ -110,6 +110,76 @@ known good output of "Hello World!".
 
 .. todo:: Add (or link to) more detailed examples of bare-metal and rocc workloads.
 
+
+Using the same directory structure for both bare-metal and buildroot
+-----------------------------------------------------------------------
+In this step, we will see how we can use the same directory structure for both 
+baremetal and buildroot.
+
+
+First, we will build a "Hello World" binary for buildroot. 
+Lets take a look at ``example-workloads/example-br.json``.
+
+.. include:: example-workloads/example-br.json
+   :code: json
+
+First you will see the ``name`` field set to ``example-br``. This indicates that 
+the directory that we will be working on. Next, you will see that our 
+base workload is set to buildroot by setting ``base`` to ``br-base.json``.
+
+The ``overlay`` field indicates the directory to overlay on the rootfs 
+which is under ``example-br/overlay``. That is, you will see a ``/root/hello-world``
+directory once you boot linux with this built workload. The files under ``overlay/root/hello-world/``
+are also copied into the file system. One caveat is that the overlay directory structure 
+has to match the rootfs directory structure.
+
+The ``host-init`` field indicates the script to run while building the workload.
+If you look into ``example-br/host-init.sh`` you can see that it runs a ``make``
+command to cross-compile our "Hello World" binary.
+
+.. include:: example-workloads/example-br/host-init.sh
+   :code: bash
+
+Finally the workload consists of individual ``jobs``. The ``outputs`` field
+specifies the files that you want to copy out from the file system after the
+simulation has ended. ``run`` sets the script to run on the target once the
+simulation starts. If you take a look into ``example-br/run.sh``, you can see
+that it is executing our "Hello World" binary and terminating the simulation by
+runnint ``poweroff``.
+
+.. include:: example-workloads/example-br/run.sh
+   :code: bash
+
+
+We are ready to build and install the workload for FireSim!
+
+::
+
+  ./marshal -v build example-workloads/example-br.json
+  ./marshal -v install example-workloads/example-br.json
```

Comment:
I would add this as a `.. note::` since FireMarshal is separate from FireSim in the documentation.